drake/systems/test/testDrakeSystemFrames.cpp

```diff
@@ -0,0 +1,39 @@
+
+#include <iostream>
+#include "DrakeSystem.h"
+
+using namespace std;
+
+int main(int argc, char* argv[]) {
+
+  shared_ptr<DrakeSystem> test_system1( new DrakeSystem("MySystem",
```

Comment:
yup.  thought I had caught them all.

> On Aug 22, 2015, at 10:12 PM, Twan Koolen <notifications@github.com> wrote:
> 
> In drake/systems/test/testDrakeSystemFrames.cpp <https://github.com/RobotLocomotion/drake/pull/1274#discussion_r37701922>:
> 
> > @@ -0,0 +1,39 @@
> > +
> > +#include <iostream>
> > +#include "DrakeSystem.h"
> > +
> > +using namespace std;
> > +
> > +int main(int argc, char* argv[]) {
> > +
> > +  shared_ptr<DrakeSystem> test_system1( new DrakeSystem("MySystem",
> make_shared is shorter and preferred because it handles exceptions in the constructor better.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/RobotLocomotion/drake/pull/1274/files#r37701922>.
>